ParabolInc/action
# Signup page

Fixes #1757.

To test:
- Check out `signup-page` locally
- Flip `newSignIn` to `true` in `releaseFlags.js`
- Start the server
- Visit the homepage
- Click on "Sign Up"
- Option 1: Sign up via third party
  - Click "Sign up with Google"
  - Get redirected to Google to sign up
  - Get redirected back to the app, and end up in the Welcome Wizard
- Option 2: Sign up via email/password
  - Enter an email and password
  - Validation works as expected
  - On submit, the form gets disabled so as to prevent double submits
    - This works on the sign-in page as well
  - End up in the Welcome Wizard
